modules/ob-uninstalling-the-builds-operator.adoc

```diff
@@ -0,0 +1,20 @@
+// Module included in the following assemblies:
+//
+// builds/uninstalling-openshift-builds.adoc
+
+:_content-type: PROCEDURE
+[id='ob-uninstalling-the-builds-operator_{context}']
+= Uninstalling the {builds-title} Operator
+
+You can uninstall the {builds-title} Operator by using the *Administrator* perspective in the web console.
+
```

Comment:
As above, if cluster admin permissions are required it should be listed as a prerequisite.